android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -1236,6 +1281,40 @@ public void run() {
     }, delay);
   }
 
+  protected void showHelpBubble() {
+    String hintText = context.getString(R.string.help_bubble_text);
+
+    // signalHelpBubbleDismissal - defined in android-host.js, gives a helpBubbleDismissed signal.
+
+    // TODO:  ensure it's called against the 'main' thread.
```

Comment:
Need to ensure this TODO is not forgotten

Reply:
True.  Note that I placed it here because I saw a corresponding error occur once during development.  Thought that was interesting, since "shouldn't the code be doing this already?"